Application/Frontend/components/FriendsColumn/Fetch_chat.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+function ChatOwnerFinder() {
+  const [channelKey, setChannelKey] = useState('');
+  const [owner, setOwner] = useState('');
+  const [error, setError] = useState('');
+
+  const fetchChatOwner = async () => {
+    setError(''); // Reset error message
+    try {
+      const response = await fetch('/api/get_chat', { // Adjust with your API endpoint
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ channelKey }),
+      });
+
+      if (!response.ok) {
+        throw new Error('Failed to fetch chat owner');
+      }
+
+      const data = await response.json();
+      if (data.owner) {
+        setOwner(data.owner);
+      } else {
+        setOwner('');
+        setError('No owner found for this channelKey');
+      }
+    } catch (err) {
+      if (err instanceof Error) {
+        setError(err.message);
+      } else {
+        setError('An error occurred while fetching the chat owner');
+      }
+    }
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    fetchChatOwner();
+  };
+
+  return (
+    <div>
```

Comment:
This is just raw HTML so you are not using any of the components from the Mantine UI library. Please modify this as your raw HTML will not only look bad in our application but won't work with any of our dynamic features like the theme toggle. For example, replace `button` with `Button` and `div` with a container like `Group`, `Stack` or `Container` from Mantine to leverage the excellent styling property of the UI components from that library.